src/style/evaluation_parameters.js

```diff
@@ -1,16 +1,26 @@
 // @flow
 
 const ZoomHistory = require('./zoom_history');
+const {isStringRenderable} = require('../util/script_detection');
+const rtlTextPlugin = require('../source/rtl_text_plugin');
+
+function isRenderable(str: string): boolean {
+    return isStringRenderable(str, rtlTextPlugin.isLoaded());
+}
 
 class EvaluationParameters {
     zoom: number;
+    isRenderable: (string) => boolean;
     now: number;
     fadeDuration: number;
     zoomHistory: ZoomHistory;
     transition: TransitionSpecification;
 
+    // address in review: options may also be another EvaluationParameters to copy, see CrossFadedProperty.possiblyEvaluate
+    // Is this necessary? If so, should it be explicitly supported?
     constructor(zoom: number, options?: *) {
         this.zoom = zoom;
+        this.isRenderable = isRenderable;
```

Comment:
`isRenderable` can just be defined directly as a method on `EvaluationParameters` (instead of declaring the `isRenderable:` property above and assigning to `this.isRenderable`)

Reply:
Good point, changed.